ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -35,14 +37,26 @@ classes:
                 type: lines
             uuid:
                 grid_display: False
+            harddisk_id:
+                grid_display: False
+                details_display: False
+                index_type: field
+            harddisk:
+                label: Hard Disk
+                api_only: true
+                api_backendtype: method
+                type: entity
+
 
     VolumeGroup:
         label: Volume Group
-        impacts:
-            - logicalVolumes
-            - snapshotVolumes
-        impacted_by:
-            - physicalVolumes
+        dynamicview_views: [service_view]
+        dynamicview_relations:
+            impacts:
+                - logicalVolumes
+                - snapshotVolumes
```

Comment:
There's no need to have the VolumeGroup directly impact its snapshotVolumes if it also impacts its logicalVolumes which then impact their snapshotVolumes.